### PR TITLE
Fixed breaking older patterns

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR fixes breaking older patterrns

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: templates/oam/workloads/1.10.0/destinationrule.istio.meshery.layer5.io_definition.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+ "kind": "WorkloadDefinition",
+ "apiVersion": "core.oam.dev/v1alpha1",
+ "metadata": {
+  "name": "DestinationRule.Istio",
+  "creationTimestamp": null
+ },
+ "spec": {
+  "definitionRef": {
+   "name": "destinationrule.Istio.meshery.layer5.io"
```
Comment (leecalcote) [inline templates/oam/workloads/1.10.0/destinationrule.istio.meshery.layer5.io_definition.json:10]: Inconsistency in casing here. Should component be lowercase?

Comment (Revolyssup) [reply templates/oam/workloads/1.10.0/destinationrule.istio.meshery.layer5.io_definition.json:10]: No this is fine. This is just to generate the file name. Plus this is consistent with metadata.name.

Comment (leecalcote) [reply templates/oam/workloads/1.10.0/destinationrule.istio.meshery.layer5.io_definition.json:10]: ok

**Comment by leecalcote** [comment]
Failing integration tests is to be expected.

**Pull request merged**